.editorconfig

```diff
@@ -0,0 +1,7 @@
+# editorconfig.org
+root = true
+
+[*.md]
+# trailing whitespaces are converted to <br>
+trim_trailing_whitespace = false
+insert_final_newline = false
```

Comment:
I'm not entirely sure if we want this. Personally, I love the prettier markdown formatting, and that would allow us to easily update all markdown files. I also disagree with both suggested markdown formatting options :D

Reply:
@ErikSchierboom so would you prefer to use [prettier](https://github.com/prettier/prettier) to format all markdown files as a GitHub action? Prettier offers a more heavy handed approach toward the same goal I'm after: file formatting consistency.

Reply:
Personally: yes, I really like prettier, but I think maybe we should discuss this first, which we didn't do. Maybe it would be good to open an issue first where we can discuss this with the community? I do agree that formatting consistency would be really nice. Being able to automatically format things would be even better.

Reply:
#863 was opened. I recognize that you missed that issue, and I think your input should be included, but as the monorepo lacks a mechanism for surfacing a meta-issue for discussion, a new issue is just going to risk being missed by other people.

Reply:
Right. Sorry that I missed that!

Reply:
I strongly disagree with `insert_final_newline = false`. Not having a final newline messes up diffs.

Reply:
Great. I'll start by updating that setting. To be honest I had copied from another source and completely agree with the point above 😄 .